domain/schema_validator.go

```diff
@@ -166,7 +177,7 @@ func (s *schemaValidator) Check(txnTS uint64, schemaVer int64, relatedTableIDs [
 	defer s.mux.RUnlock()
 	if !s.isStarted {
 		log.Infof("[domain-ddl] the schema validator stopped before checking")
-		return ResultFail
```

Comment:
When the returned result is `ResultFail`, the caller will sleep a while and retry.
After you change it to `ResultStopped`, what will happen? @winkyao

Reply:
I know the caller will retry when the error is infoschemaChanged, but if the schemavalidator is stopped, shall we retry or return the error immediately?

Reply:
Done, we now will retry ErrSchemaValidatorStopped in session.isRetryableError. Please take a look. @tiancaiamao

Reply:
Fine